Jenkinsfile

```diff
@@ -0,0 +1,48 @@
+#!/usr/bin/env groovy
```

Comment:
I guess we can't really know that this works until we get a Jenkins - I'm happy to merge it and see.